src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    private Random random = new Random(100);
```

Comment:
Avoid using a fixed seed for the Random object in production code unless you want to produce a predictable sequence of numbers (e.g., for testing). If you want to generate different sequences of random numbers on each execution, you should use the no-argument constructor.